src/main/java/org/opentripplanner/netex/mapping/StopAndStationMapper.java

```diff
@@ -87,7 +87,7 @@ void mapParentAndChildStops(final Collection<StopPlace> stopPlaces) {
         // were deleted in never versions of the StopPlace
         for (StopPlace stopPlace : stopPlaceAllVersions) {
             for (Quay quay : listOfQuays(stopPlace)) {
-                addNewStopToParentIfNotPresent(quay, station, fareZones);
+                addNewStopToParentIfNotPresent(quay, station, fareZones, stopPlace);
```

Comment:
Here you are passing in a complex type (stopPlace) and passes it down the call stack until you extract one single pice of information from it. Is it better to extract the mode here or at least in the `addNewStopToParentIfNotPresent`?

If the mode is changed do we want the mode from the most resent StopPlace or the old one. I think it make more sense to use the "most resent stop-place"; hence extract the mode from `selectedStopPlace` and pass it in the same way as fareZones.

Reply:
Minor thing - arg order: `quay, station, parentStationMode, fareZones`

Reply:
We want to keep the dependencies low, e.g. it is much easier to write a unit test if we need to pas in a mode, instead of a StopPlace.